spec/models/mou_signature_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "rails_helper"
+
+RSpec.describe MouSignature, type: :model do
+  let(:organisation) { create(:organisation) }
+  let(:user) { create(:user, organisation:) }
+  let(:mou_signature) do
+    described_class.create!(
+      user:,
+      organisation:,
+      agreed_at: Time.zone.now,
+    )
+  end
+
+  it "setting user to nil triggers a DB constraint" do
```

Comment:
Nitpick: these specs are supposed to read something like:

`it "sets user to nil and triggers a DB constraint"` 

or

```
context "setting the user to nil" do
  it "triggers a DB constraint" do
    ...
  end
end
```

one to be aware of for future